patches/0001-archive-tar-do-not-populate-user-group-names.patch

```diff
@@ -0,0 +1,79 @@
+From bc0de86b495ae014b209431ed7cb90fc3d5e4d1f Mon Sep 17 00:00:00 2001
+From: Kir Kolyshkin <kolyshkin@gmail.com>
+Date: Mon, 9 Apr 2018 15:58:40 -0700
+Subject: [PATCH] archive/tar: do not populate user/group names
+
+This reverts part of commit 29a18899379c ("archive/tar: populate
+uname/gname/devmajor/devminor in FileInfoHeader"). The reason is
+using os/user functions to resolved uids/gids to names breaks
+the static build for Linux/glibc case (the resulting binary panics
+on NULL pointer dereference).
+
+For much more details, see https://github.com/golang/go/issues/23265
+
+This is ultimately fixed in Go 1.11 (provided that "osusergo" build tag
+is set (see https://github.com/golang/go/commit/62f0127d81). In the
+meantime, let's use this fork.
```

Comment:
I'm guessing this description isn't accurate given we're patching `archive/tar` again in a post-1.11 Go version?

Reply:
We reused the same commit that we used in the past (but at the time for a different reason)

Reply:
Right, that makes sense, but shouldn't we update the description?  From a naive reading it looks like this patch existing again is a mistake, since it's "fixed" in 1.11+. :sweat_smile:

Is there an issue filed on https://github.com/golang/go we could link to for further discussion?  Are they open to a knob for disabling this behavior in a future Go version that could re-deprecate this patch in the future?

Reply:
@kolyshkin @tonistiigi ^^

Reply:
There is no issue for making this behavior configurable afaik

Reply:
No, AFAIK no one filed it. Makes sense to do so (at least to start a conversation). I'm pretty busy atm though :(

Reply:
@tonistiigi could you open a ticket? You're probably better at describing the problem than I am